src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -297,3 +299,55 @@ def registry_sku_validation(registry_name, resource_group_name=None, message=Non
         resource_group_name = get_resource_group_name_by_resource_id(arm_resource.id)
 
     return arm_resource, resource_group_name
+
+
+def validate_sku_update(sku_parameter):
+    """Validates a registry SKU update parameter.
+    :param object sku_parameter: The registry SKU update parameter
+    """
+    if sku_parameter is None:
+        return
+
+    if isinstance(sku_parameter, dict):
+        if 'name' not in sku_parameter or sku_parameter['name'] not in MANAGED_REGISTRY_SKU:
+            _invalid_sku_update()
+    elif isinstance(sku_parameter, Sku):
+        if sku_parameter.name not in MANAGED_REGISTRY_SKU:
+            _invalid_sku_update()
+    else:
+        _invalid_sku_update()
+
+
+def _invalid_sku_update():
+    raise CLIError(
```

Comment:
This method is used in the same file by the method `validate_sku_update` above it, or did you mean both methods should be moved to `custom.py` since that is where they are called?